shared/middleware/helpers/table.rb

```diff
@@ -32,9 +92,15 @@ def delete(id)
 
   def delete_all()
     items.delete
+    @metadata_dataset.delete if metadata_dataset
   end
```

Comment:
I was misremembering -- at one point in https://www.pivotaltracker.com/story/show/109992012 we wanted to remember the column types when we overwrite and import, but I think we decided not to do that for now. It should be fine to delete here.